reference/docs-conceptual/overview.md

```diff
@@ -1,206 +1,64 @@
-PowerShell is a cross-platform task automation and configuration management framework, consisting of
-a command-line shell and scripting language. Unlike most shells, which accept and return text,
-PowerShell is built on top of the .NET Common Language Runtime (CLR), and accepts and returns .NET
-objects. This fundamental change brings entirely new tools and methods for automation.
+PowerShell is a cross-platform task automation solution made up of a command-line shell, a scripting
+language, and a configuration management framework. PowerShell runs on Windows, Linux, and macOS.
 
-<!-- removing images until we can get replacements
-:::row:::
-   :::column span="":::
-      Windows
-      [![PowerShell on Windows](media/overview/windows-desktop-660.gif)](media/overview/windows-desktop.gif#lightbox)
-      [Install on Windows](install/installing-powershell-core-on-windows.md)
-   :::column-end:::
-   :::column span="":::
-      Linux
-      [![PowerShell on Linux](media/overview/linux-desktop-660.gif)](media/overview/linux-desktop.gif#lightbox)
-      [Install on Linux](install/installing-powershell-core-on-linux.md)
-   :::column-end:::
-   :::column span="":::
-      macOS
-      [![PowerShell on macOS](media/overview/macos-desktop-660.gif)](media/overview/macos-desktop.gif#lightbox)
-      [Install on macOS](install/installing-powershell-core-on-macos.md)
-   :::column-end:::
-:::row-end:::
--->
+## Shell
 
-## Output is object-based
+PowerShell is modern command shell that includes all of the best features of other popular shells.
+Unlike most shells that only accept and return text, PowerShell accepts and returns .NET Objects.
 
-Unlike traditional command-line interfaces, PowerShell cmdlets are designed to deal with objects.
-An object is structured information that is more than just the string of characters appearing on the
-screen. Command output always carries extra information that you can use if you need it.
+Features
 
-If you've used text-processing tools to process data in the past, you'll find that they behave
-differently when used in PowerShell. In most cases, you don't need text-processing tools to extract
-specific information. You directly access portions of the data using standard PowerShell object
-syntax.
+- Robust command-line [history][]
+- Tab completion and command prediction (See [about_PSReadLine][])
+- Supports command and parameter [aliases][]
+- [Pipeline][] for chaining commands
+- In console [help][] system similar to Unix `man` pages
 
-## The command family is extensible
+## Scripting language
 
-Interfaces such as `cmd.exe` don't provide a way for you to directly extend the built-in command
-set. You can create external command-line tools that run in `cmd.exe`. But these external tools
-don't have services, such as Help integration. `cmd.exe` doesn't automatically know that these
-external tools are valid commands.
+PowerShell is built on the .NET Common Language Runtime (CLR). All inputs and outputs are .NET
```

Comment:
I was talking to @theJasonHelmick about this as well. He suggested a "call to action", at the bottom maybe, where we link to learning resources and how to get started.

Reply:
In line with this recommendation, it might be good to include the "why?" of the scripting. E.g. "as a scripting language, PowerShell is commonly used for automating the management of systems; build, test, and deployment scripts (often in CI/CD environments; and more...." 

I don't like my language there, but something like that.

Reply:
Updated